package.json

```diff
@@ -58,7 +58,7 @@
     "js-yaml": "^4.1.0",
     "lodash": "^4.17.21",
     "screwdriver-config-parser": "^10.0.0",
-    "screwdriver-data-schema": "^23.0.4",
+    "screwdriver-data-schema": "^23.4.0",
```

Comment:
Would be available after https://github.com/screwdriver-cd/data-schema/pull/569 is merged